### PR TITLE
fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S ./share/locale,./thirdparty -L ba,cann,clas,dur,foto,iff,nd,ois,ot,pres,possibile,snaped,strack,tage,te,uint,thru,valu`

### CODE REVIEW

File: libmscore/cmd.cpp [hunk 1]
```diff
@@ -847,7 +847,7 @@ Fraction Score::makeGap(Segment* segment, int track, const Fraction& _sd, Tuplet
       Q_ASSERT(_sd.numerator());
 
       Measure* measure = segment->measure();
-      Fraction akkumulated;
+      Fraction accumulated;
```
Comment (luzpaz) [inline libmscore/cmd.cpp]: source code change

File: effects/compressor/compressor.h [hunk 2]
```diff
@@ -67,7 +67,7 @@ class Compressor : public Effect
       unsigned int count;
       float as[A_TBL];
 
-      // The blanace between the RMS and peak envelope followers.
+      // The blance between the RMS and peak envelope followers.
```
Comment (Jojo-Schmitz) [inline effects/compressor/compressor.h]: Wrong before and after, needs to be `balance'

File: libmscore/cmd.cpp [hunk 3]
```diff
@@ -872,7 +872,7 @@ Fraction Score::makeGap(Segment* segment, int track, const Fraction& _sd, Tuplet
                   Fraction td(tick2 - seg->tick());
                   if (td > sd)
                         td = sd;
-                  akkumulated += td;
+                  accumulated += td;
```
Comment (luzpaz) [inline libmscore/cmd.cpp]: source code change

File: libmscore/cmd.cpp [hunk 4]
```diff
@@ -884,7 +884,7 @@ Fraction Score::makeGap(Segment* segment, int track, const Fraction& _sd, Tuplet
                   Fraction td(seg->tick() - nextTick);
                   if (td > sd)
                         td = sd;
-                  akkumulated += td;
+                  accumulated += td;
```
Comment (luzpaz) [inline libmscore/cmd.cpp]: source code change

Comment (luzpaz) [reply libmscore/cmd.cpp]: There are several more instances of the same typo fix below

File: libmscore/pos.cpp [hunk 5]
```diff
@@ -586,17 +586,17 @@ void Pos::downSnap(int raster)
       setTick(sig->raster1(tick(), raster));
       }
 
-Pos Pos::snaped(int raster) const
+Pos Pos::snapped(int raster) const
```
Comment (luzpaz) [inline libmscore/pos.cpp]: source code change

File: libmscore/pos.cpp [hunk 6]
```diff
@@ -586,17 +586,17 @@ void Pos::downSnap(int raster)
...
       {
       return Pos(tempo, sig, sig->raster(tick(), raster));
       }
 
-Pos Pos::upSnaped(int raster) const
+Pos Pos::upSnapped(int raster) const
```
Comment (luzpaz) [inline libmscore/pos.cpp]: source code change

File: libmscore/scorediff.cpp [hunk 7]
```diff
@@ -211,15 +211,15 @@ int MscxModeDiff::adjustSemanticsMscxOneDiff(std::vector<TextDiff>& diffs, int i
       int iScore;
       switch(diff->type) {
             case DiffType::EQUAL:
-                  /* FALLTROUGH */
+                  /* FALLTHROUGH */
```
Comment (Jojo-Schmitz) [inline libmscore/scorediff.cpp]: Maybe use  `Q_FALLTHROUGH();` here instead?

File: libmscore/pos.cpp [hunk 8]
```diff
@@ -586,17 +586,17 @@ void Pos::downSnap(int raster)
...
       {
       return Pos(tempo, sig, sig->raster2(tick(), raster));
       }
 
-Pos Pos::downSnaped(int raster) const
+Pos Pos::downSnapped(int raster) const
```
Comment (luzpaz) [inline libmscore/pos.cpp]: source code change

File: libmscore/pos.h [hunk 9]
```diff
@@ -68,9 +68,9 @@ class Pos {
       void snap(int);
       void upSnap(int);
       void downSnap(int);
-      Pos snaped(int) const;
-      Pos upSnaped(int) const;
-      Pos downSnaped(int) const;
+      Pos snapped(int) const;
+      Pos upSnapped(int) const;
+      Pos downSnapped(int) const;
```
Comment (luzpaz) [inline libmscore/pos.h]: source code change

File: libmscore/scorediff.cpp [hunk 10]
```diff
@@ -211,15 +211,15 @@ int MscxModeDiff::adjustSemanticsMscxOneDiff(std::vector<TextDiff>& diffs, int i
...
             case DiffType::REPLACE:
                   // TODO: split a REPLACE diff, though they should not be here
                   return index;
             case DiffType::INSERT:
                   iScore = 1;
                   break;
             case DiffType::DELETE:
-                  /* FALLTROUGH */
+                  /* FALLTHROUGH */
```
Comment (Jojo-Schmitz) [inline libmscore/scorediff.cpp]: dito

File: libmscore/scorediff.cpp (hunk 7 above)
Comment (luzpaz) [reply libmscore/scorediff.cpp]: Will change. BTW, there are also instances of `FALLTHROUGH` in `libmscore/page.cpp` & `libmscore/pitchspelling.cpp`  
Shall I change those as well ?

File: effects/compressor/compressor.h (hunk 2 above)
Comment (luzpaz) [reply effects/compressor/compressor.h]: ooops, fixed in 7d0f10a

**Comment by Howard-C** [comment]
Now please squash (via rebase) these commits into one.

File: libmscore/scorediff.cpp (hunk 7 above)
Comment (Jojo-Schmitz) [reply libmscore/scorediff.cpp]: Not sure (I just brought that up in the developers' Telegram chat), at least those are not typos, and those in page.cpp won't work with a trailing `;`

Comment (Jojo-Schmitz) [reply libmscore/scorediff.cpp]: Actually in page.cpp it doesn't work with `;` (as that would change the semanitic) and on MSVC not without either (compiler error C2143). So better leave it as is. Changing pitchspelling.cpp should be OK though.

File: libmscore/scorediff.cpp (hunk 10 above)
Comment (luzpaz) [reply libmscore/scorediff.cpp]: Done: 44699f8

**Comment by luzpaz** [comment]
@Howard-C I will be happy to once review is complete.

File: libmscore/scorediff.cpp (hunk 7 above)
Comment (luzpaz) [reply libmscore/scorediff.cpp]: @Jojo-Schmitz done in c975b7d

File: libmscore/pitchspelling.cpp [hunk 11]
```diff
@@ -684,11 +684,11 @@ void Score::spellNotelist(std::vector<Note*>& notes)
                         case 3:
                               k = end - start - 3;
                               changeAllTpcs(notes[end-3], tab[(notes[end-3]->pitch() % 12) * 2 + ((opt & (1<<k)) >> k)]);
-                              // FALLTHROUGH
+                              // Q_FALLTHROUGH();
```
Comment (Jojo-Schmitz) [inline libmscore/pitchspelling.cpp]: No, not as a comment, you need to drop the `// ` here

File: libmscore/scorediff.cpp [hunk 12]
```diff
@@ -211,15 +211,15 @@ int MscxModeDiff::adjustSemanticsMscxOneDiff(std::vector<TextDiff>& diffs, int i
       int iScore;
       switch(diff->type) {
             case DiffType::EQUAL:
-                  /* FALLTHROUGH */
+                  /* Q_FALLTHROUGH(); */
```
Comment (Jojo-Schmitz) [inline libmscore/scorediff.cpp]: same here

Comment (Jojo-Schmitz) [reply libmscore/scorediff.cpp]: Actually here, as there are no statements between those 2 pairs of fall thouigh cases, nothing is needed, the compiler won't warn (and that's most probably the reason why those typos went unnoticed), so best remove those 2 lines entirely

File: libmscore/pitchspelling.cpp (hunk 11 above)
Comment (luzpaz) [reply libmscore/pitchspelling.cpp]: just for `case 3:` or should I also modify `case 2:` this way as well?

Comment (Jojo-Schmitz) [reply libmscore/pitchspelling.cpp]: both in this file

File: libmscore/scorediff.cpp (hunk 12 above)
Comment (Jojo-Schmitz) [reply libmscore/scorediff.cpp]: and both in this file too

Comment (luzpaz) [reply libmscore/scorediff.cpp]: @Jojo-Schmitz so just to confirm, you're asking me to delete both `/* Q_FALLTHROUGH(); */` lines ?

Comment (Jojo-Schmitz) [reply libmscore/scorediff.cpp]: yes

Comment (Jojo-Schmitz) [reply libmscore/scorediff.cpp]: removing unneeded stuff is better than fixing it ;-)

File: mscore/startcenter.cpp [hunk 13]
```diff
@@ -75,7 +75,7 @@ Startcenter::Startcenter(QWidget* parent)
 
             horizontalLayout->addWidget(_webView);
             
-            //workaround for the crashes sometimes happend in Chromium on macOS with Qt 5.12
+            //workaround for the crashes sometimes happpening in Chromium on macOS with Qt 5.12
```
Comment (Jojo-Schmitz) [inline mscore/startcenter.cpp]: one p too much ;-)

File: mscore/network/loginmanager.cpp [hunk 14]
```diff
@@ -373,7 +373,7 @@ static void clearHttpCacheOnRenderFinish(QWebEngineView* webView)
       QWebEnginePage* page = webView->page();
       QWebEngineProfile* profile = page->profile();
 
-      // workaround for the crashes sometimes happend in Chromium on macOS with Qt 5.12
+      // workaround for the crashes sometimes happenning in Chromium on macOS with Qt 5.12
```
Comment (Jojo-Schmitz) [inline mscore/network/loginmanager.cpp]: one n too much

File: libmscore/pitchspelling.cpp (hunk 11 above)
Comment (luzpaz) [reply libmscore/pitchspelling.cpp]: Done. e2ca31e

File: libmscore/scorediff.cpp (hunk 12 above)
Comment (luzpaz) [reply libmscore/scorediff.cpp]: Indeed :) e2ca31e

File: mscore/startcenter.cpp (hunk 13 above)
Comment (luzpaz) [reply mscore/startcenter.cpp]: yikes, thanks. I'll update to codespell dictionary to catch this. e2ca31e

File: mscore/network/loginmanager.cpp (hunk 14 above)
Comment (luzpaz) [reply mscore/network/loginmanager.cpp]: yikes, thanks. I'll update to codespell dictionary to catch this. e2ca31e

**Comment by Jojo-Schmitz** [comment]
I believe it is time to squash.

**Comment by luzpaz** [comment]
Squashed a4dee11

**Comment by luzpaz** [comment]
@Jojo-Schmitz I found 2 more typos. Can I add to this PR for review ?

**Comment by Jojo-Schmitz** [comment]
Sure

File: share/styles/chords_jazz.xml [hunk 15]
```diff
@@ -146,7 +146,7 @@
 
     </font>
 
-  <font class="modifer" family="MuseJazz Text">
+  <font class="modifier" family="MuseJazz Text">
```
Comment (luzpaz) [inline share/styles/chords_jazz.xml]: I checked and saw that there wasn't an actual class named `modifer` but I did find a whole lot of classes named `modifiers`

File: mscore/prefsdialog.ui [hunk 16]
```diff
@@ -688,7 +688,7 @@
            </widget>
           </item>
           <item row="1" column="2">
-           <widget class="QLabel" name="iconHeigthLabel">
+           <widget class="QLabel" name="iconHeightLabel">
```
Comment (luzpaz) [inline mscore/prefsdialog.ui]: This looks to be a unique source typo as well.

File: share/styles/chords_jazz.xml (hunk 15 above)
Comment (Jojo-Schmitz) [inline share/styles/chords_jazz.xml]: Looks right to me, but is a code change. @MarcSabatella ?

File: mscore/importmxmlpass2.cpp [hunk 17]
```diff
@@ -1755,7 +1755,7 @@ static void handleBeamAndStemDir(ChordRest* cr, const Beam::Mode bm, const Direc
             static_cast<Chord*>(cr)->setStemDirection(sd);
             cr->setBeamMode(Beam::Mode::AUTO);
             }
-      // terminate the currect beam and add to the score
+      // terminate the correct beam and add to the score
```
Comment (luzpaz) [inline mscore/importmxmlpass2.cpp]: wasn't sure if this was `current` or `correct`, I chose the latter.

File: mscore/prefsdialog.ui (hunk 16 above)
Comment (Jojo-Schmitz) [inline mscore/prefsdialog.ui]: This is a code change, but OK as long as that name isn't referenced elsewhere

File: share/styles/chords_jazz.xml (hunk 15 above)
Comment (luzpaz) [reply share/styles/chords_jazz.xml]: > I checked and saw that there wasn't an actual class named modifer but I did find a whole lot of classes named modifier

File: mscore/prefsdialog.ui (hunk 16 above)
Comment (luzpaz) [reply mscore/prefsdialog.ui]: > This looks to be a unique source typo as well.

Indeed, that's why i made a comment on it (but you beat me to it :smiley:)

File: mscore/importmxmlpass2.cpp (hunk 17 above)
Comment (Jojo-Schmitz) [reply mscore/importmxmlpass2.cpp]: I believe it shgould be `current` though

Comment (luzpaz) [reply mscore/importmxmlpass2.cpp]: Done. f50dbab

File: share/styles/chords_jazz.xml (hunk 15 above)
Comment (MarcSabatella) [reply share/styles/chords_jazz.xml]: It's not a class in the C++ sense, but an XML attribute on the font tag, and this typo probably causes some things about chord symbol rendering not to work correctly, so good catch!  Now I just need to make sure whatever change is caused by this (rendering of certain chord symbols in Jazz style) is good, so let me test a little and get back to you.

Comment (MarcSabatella) [reply share/styles/chords_jazz.xml]: Yes, this actually fixes a bug: the parentheses and comma in C7(b9,b5) do not respond correctly to the "modifier" scaling & offset settings in Format / Style / Chord Symbols.  So this change is not just cosmetic, it is needed in order to address a real issue.

Are there other changes in here like that, or is it mostly non-functional changes?  I do want to make sure this gets fixed, and am trying to decide it is worth submitting a separate PR for just this.

**Comment by Howard-C** [comment]
BTW, is "partitur" really a English term? I've seen that in the source code of Choose Instrument dialog and wonder if it can be changed to a English term (if it's not).

**Comment by luzpaz** [comment]
> BTW, is "partitur" really a English term? 

@Howard-C Per [merriam-webster](https://www.merriam-webster.com/dictionary/partitur) it is indeed.

File: mscore/prefsdialog.ui (hunk 16 above)
Comment (luzpaz) [reply mscore/prefsdialog.ui]: @MarcSabatella here is a source change :point_up_2:

**Comment by Howard-C** [comment]
> > BTW, is "partitur" really a English term?
> 
> @Howard-C Per [merriam-webster](https://www.merriam-webster.com/dictionary/partitur) it is indeed.

Wow, you learn something new every day!

File: mscore/prefsdialog.ui (hunk 16 above)
Comment (MarcSabatella) [reply mscore/prefsdialog.ui]: Yes, but labels are almost never referenced by the code.  If you didn't do a search on any such strings already, do so, but I'm guessing there won't be an issue.

**Comment by MarcSabatella** [comment]
Partitur is an English word the way "doth" or "thee" are :-).  Definitely not something that should appear in the UI, but it doesn't both me to occasionally see foreign-looking words in the source.

File: mscore/prefsdialog.ui (hunk 16 above)
Comment (Jojo-Schmitz) [reply mscore/prefsdialog.ui]: I did and it is not

Comment (luzpaz) [reply mscore/prefsdialog.ui]: I did as well. So that does it then, squash + rebase time ?

Comment (Jojo-Schmitz) [reply mscore/prefsdialog.ui]: yes

**Comment by luzpaz** [comment]
pushed and rebased. Should be ready to merge

**Comment by Jojo-Schmitz** [comment]
Expect a Travis failure... not your fault ;-)

**Comment by Jojo-Schmitz** [comment]
Hmm, **that** Travis failure was not expected (but still not your fault)

**Comment by Howard-C** [comment]
Now you can push again and make Travis build pass ;-)

**Comment by luzpaz** [comment]
Rebased and force pushed.

**Pull request merged**